optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -53,11 +53,16 @@
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-core</artifactId>
       <exclusions>
-        <!-- Clash with jakarta.activation:jakarta.activation-api -->
+        <!-- Duplicates classes of jakarta.activation:jakarta.activation-api -->
         <exclusion>
           <groupId>javax.activation</groupId>
           <artifactId>javax.activation-api</artifactId>
         </exclusion>
+        <!-- Duplicates classes of jakarta.xml.bind:jakarta.xml.bind-api -->
+        <exclusion>
+          <groupId>javax.xml.bind</groupId>
+          <artifactId>jaxb-api</artifactId>
+        </exclusion>
```

Comment:
Can we avoid duplicating the exclusions by doing them in the optaplanner-build-parent?

Reply:
Got the same question from Lukas. The answer is both yes and no:

The vast majority of the exclusions come from `io.quarkus` artifacts. If we define the exclusions in dependency management, we have to define a version of every such `io.quarkus` artifact. Effectively, we duplicate (and potentially) override the version coming from quarkus BOM.

For this reason, I would rather keep the exclusions at individual dependencies. Chances are there will be much less of them after #824 is merged.